test/integration-test/bpf/kconfig.bpf.c

```diff
@@ -3,11 +3,24 @@
 #include <bpf/bpf_helpers.h>
 // clang-format on
 
+// CONFIG_BPF=y => 1
 extern unsigned int CONFIG_BPF __kconfig;
+// CONFIG_PANIC_TIMEOUT=0 => 0
+extern unsigned int CONFIG_PANIC_TIMEOUT __kconfig;
+// CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=120
+extern unsigned int CONFIG_DEFAULT_HUNG_TASK_TIMEOUT __kconfig;
 
 SEC("xdp")
 int kconfig(struct xdp_md *ctx) {
-  if (!CONFIG_BPF) {
+  if (CONFIG_BPF != 1) {
+    return XDP_DROP;
+  }
+
+  if (CONFIG_PANIC_TIMEOUT != 0) {
+    return XDP_DROP;
+  }
+
+  if (CONFIG_DEFAULT_HUNG_TASK_TIMEOUT != 120) {
     return XDP_DROP;
   }
 
```

Comment:
Testing more than one config was something I initially wanted to do and forgot, thanks for doing that :smile: